config/config_test.go

```diff
@@ -231,37 +233,41 @@ func TestNewHorusecConfig(t *testing.T) {
 		assert.NoError(t, os.Setenv(config.EnvEnableShellCheck, "true"))
 		assert.NoError(t, os.Setenv(config.EnvCustomRulesPath, "test"))
 		assert.NoError(t, os.Setenv(config.EnvEnableInformationSeverity, "true"))
-		assert.NoError(t, os.Setenv(config.EnvShowVulnerabilitiesTypes, fmt.Sprintf("%s, %s", vulnerability.Vulnerability.ToString(), vulnerability.RiskAccepted.ToString())))
```

Comment:
I remove this SetEnv to assert that when calling [factoryParseInputToSliceString](https://github.com/ZupIT/horusec/blob/develop/config/config.go#L621) with default array value on input we parse correctly.